lib/ansible/module_utils/web_infrastructure/ansible_tower/api.py

```diff
@@ -0,0 +1,124 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
We already have `module_utils/ansible_tower.py` should that be moved here @AlanCoding ?